ipatests/test_integration/test_cert.py

```diff
@@ -216,6 +216,52 @@ def test_getcert_list_profile_using_subca(self, test_subca_certs):
             raise AssertionError("certmonger request is "
                                  "in state {}". format(status))
 
+    def test_certmonger_rekey_option(self):
+        """Test certmonger rekey command works fine
+
+        Certmonger's rekey command was throwing an error as
+        unrecognized command. Test is to check if it is working fine.
+
+        related: https://bugzilla.redhat.com/show_bug.cgi?id=1249165
+        """
+        result = self.master.run_command([
+            'ipa-getcert', 'request',
+            '-f', '/etc/pki/tls/certs/test_rekey.pem',
+            '-k', '/etc/pki/tls/private/test.key',
+            '-K', 'test/{}'.format(self.master.hostname)])
+        request_id = re.findall(r'\d+', result.stdout_text)
```

Comment:
You can use get_certmonger_fs_id() from ipatests.test_integration.test_certs for this regex instead.

Reply:
Initially I used it but it returns `request_id[1]` and we are interested in `request_id[0]` here

Reply:
Ok, right, you're using the output of getcert. Another way to do this is to use -I <name> to specify the nickname you want to use for this cert, then you don't have to regex the output to find the certmonger-generated id. What you are doing is fine.